.chronus/changes/use_diagnostics-2024-1-14-18-46-55.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: breaking
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+include list of diagnostics when creating sdk types
```

Comment:
can you describe the breaking change a little more on what they might have to change